state_synchronizer/src/coordinator.rs

```diff
@@ -320,6 +328,38 @@ impl<T: ExecutorProxyTrait> SyncCoordinator<T> {
                 ));
             }
         }
+
+        let previous_version = self.known_version;
+        let chunk_size = txn_list_with_proof.get_transactions().len() as u64;
+
+        if let Err(e) = self
```

Comment:
actually no need to explicitly check for error here. It should be enough to check progress afterwards